extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -143,39 +141,32 @@ export const SendAmount = ({
 
   const calculateAvailBalance = useCallback(
     (selectedAsset: string) => {
-      let availBalance = new BigNumber("0");
+      let availBalance = "0";
       if (isToken) {
         const contractId = selectedAsset.split(":")[1];
         return getTokenBalance(tokenBalances, contractId);
       }
       if (accountBalances.balances) {
-        // take base reserve into account for XLM payments
-        const minBalance = new BigNumber(
-          (2 + accountBalances.subentryCount) * BASE_RESERVE,
-        );
-
         if (selectedAsset === "native") {
+          // take base reserve into account for XLM payments
+          const baseReserve = (2 + accountBalances.subentryCount) * 0.5;
+
           // needed for different wallet-sdk bignumber.js version
           const currentBal = new BigNumber(
             accountBalances.balances[selectedAsset].total.toFixed(),
           );
-
           availBalance = currentBal
-            .minus(minBalance)
-            .minus(new BigNumber(Number(recommendedFee)));
-
-          if (availBalance.lt(minBalance)) {
-            return "0";
-          }
```

Comment:
I believe we want to stick with the previous version here that matches master branch, it fixes a display bug for balances.